standalone/src/main/java/io/pravega/local/LocalPravegaEmulator.java

```diff
@@ -14,6 +14,12 @@
 import lombok.Getter;
 import lombok.extern.slf4j.Slf4j;
 
+/**
+ * Main entry point for Integration tests that need an in process Pravega cluster.
+ * This class is intended to be used both by internal test suites
+ * and  by test suites of applications that want to run tests against
```

Comment:
nit: an extra space before `by` and `in-process`

Reply:
@ravisharda I have corrected the extra space and the in-process word.
I can't see where to add another '.'